docker/Dockerfile

```diff
@@ -42,7 +56,6 @@ RUN set -x \
     && adduser "${BK_USER}" \
     && apt-get update \
     && apt-get install -y ca-certificates apt-transport-https \
-    && apt-get install -y --no-install-recommends openjdk-17-jdk \
```

Comment:
as I understand, this silently/without discussion downgrades jdk from 17 to 11 (included in `eclipse-temurin:11`)

Reply:
Good catch!